.gatekeeper_env

```diff
@@ -6,9 +6,9 @@ if [ "$0" = "${BASH_SOURCE[0]}" ]; then
     exit 1
 fi
 
-export GATEKEEPER_CLIENT_ID=<CLIENT ID>
-export GATEKEEPER_CLIENT_SECRET=<CLIENT SECRET>
-export GATEKEEPER_ENCRYPTION_KEY=<ENCRYPTION_KEY>
+#export GATEKEEPER_CLIENT_ID=<CLIENT ID>
+#export GATEKEEPER_CLIENT_SECRET=<CLIENT SECRET>
+#export GATEKEEPER_ENCRYPTION_KEY=<ENCRYPTION_KEY>
```

Comment:
It would be nicer to set these to insecure, and at the bottom of this file immediately check for the variables; if set to insecure, throw an error stating that they need to be set properly. This allows for immediate feedback when running `source .gatekeeper_env`.

Also, it might be wise to add this file to the gitignore, but leaving it in the repository. That will prevent developers setting the ENV vars, and committing the values accidentally.